01_install_requirements.sh

```diff
@@ -90,7 +90,7 @@ case $DISTRO in
     elif [[ $DISTRO == "rhel9" ]]; then
       # NOTE(raukadah): If a system is subscribed to RHEL subscription then
       # sudo subscription-manager identity will return exit 0 else 1.
-      if $(sudo subscription-manager identity > /dev/null 2>&1); then
+      if (sudo subscription-manager identity > /dev/null 2>&1); then
```

Comment:
Good catch, I always miss this :( Could you remove the parenthesis as well?

Reply:
Sure, done – I originally kept the parenthesis for readability only